collections/global-human-settlement-layer-ghs-built-s2.yaml

```diff
@@ -0,0 +1,89 @@
+  and settlements to monitor human presence on Earth over time.
+  The Global Human Settlement Layer GHS-BUILT-S2 is a global map of built-up areas (expressed as probabilities) at 10 m spatial resolution.
+  It was derived from a Sentinel-2 global image composite for the reference year 2018 using Convolutional Neural Networks.
+AdditionalInfoExternal:
+    Title: Additional info
+    Path: global-human-settlement-layer-ghs-built-s2/README.MD
+Image: global-human-settlement-layer-ghs-built-s2/global-human-settlement-layer-ghs-built-s2.png
+# EOBrowser: https://sentinelshare.page.link/sgSh
+Resolution: 10m
+GeographicalCoverage: Global coverage with longitude from 180°W to 180°E and latitude from 72°N to 56°S
+TemporalAvailability: Reference year 2018
+UpdateFrequency: Irregular
+BandInformation: Built-up probability [0 - 100 %]
+Contact: "[Joint Research Centre - Global Human Settlement Contact](jrc-ghsl-data@ec.europa.eu)"
+Provider: "[European Commission, Joint Research Centre (JRC)](https://ghsl.jrc.ec.europa.eu/)"
+ManagedBy: "[Sentinel Hub](https://www.sentinel-hub.com/)"
+Tags:
+  - sentinel hub
+  - raster
+  - urban mapping
+  - machine learning
+  - derived data
+  - open data
+License: |
+  [Creative Commons Attribution 4.0 International License](https://creativecommons.org/licenses/by/4.0/), find more details on use and access conditions [here](https://data.jrc.ec.europa.eu/dataset/016d1a34-b184-42dc-b586-e10b915dd863)
+LicenseType: proprietary
+LicenseUrl: https://creativecommons.org/licenses/by/4.0/
+Resources:
+  - Group: Sentinel Hub Resources
+    EndPoint: creodias.sentinel-hub.com
+    Name: Sentinel Hub
+    Role: processor
+    Type: byoc-3dbeea2c-3207-4c65-8a73-c29ce2675f89
+    CollectionId: 3dbeea2c-3207-4c65-8a73-c29ce2675f89
+    Primary: true
+  - Group: xcube Resources
+    DatasetName: CUSTOM
+    CollectionId: 3dbeea2c-3207-4c65-8a73-c29ce2675f89
+CustomScripts:
+    Title: Collection of Copernicus services data custom scripts
+    URL: https://custom-scripts.sentinel-hub.com/#copernicus_services
+Configurations :
+  - layer_name: GHS-BUILT-S2
+    evalscript_url: "https://custom-scripts.sentinel-hub.com/custom-scripts/copernicus_services/global-human-settlement-layer-ghs-built-s2/script.js"
+    mosaicking_order: mostRecent
+    upsampling: BICUBIC
+    href: https://docs.sentinel-hub.com/api/latest/api/process/
+    rel: processing-expression
+Extent:
+  spatial:
+    bbox:
+      -
+        - -180
+        - -56
+        - 180
+        - 72
+  temporal:
+    interval:
+      -
+        - '2018-01-01T00:00:00Z'
+        - '2018-12-31T00:00:00Z'
+CubeDimensions:
+  x:
+    type: spatial
+    axis: x
+    extent:
+      - -180
+      - 180
+  y:
+    type: spatial
+    axis: y
+    extent:
+      - -56
+      - 72
+  t:
+    type: temporal
+    extent:
+      - '2018-01-01T00:00:00Z'
+      - '2018-12-31T00:00:00Z'
+    step: P5D
```

Comment:
Would remove this line (it's not mandatory) as it means that there is a new image every 5 days in the specified extent (https://github.com/stac-extensions/datacube#temporal-dimension-object).